Tests/FingerprintJSTests/FingerprintJSVersionTests.swift

```diff
@@ -0,0 +1,43 @@
+import XCTest
+
+@testable import FingerprintJS
+
+final class FingerprintJSVersionTests: XCTestCase {
+    func test_givenVersionOne_whenSince_thenReturnsAllVersions() {
+        // given
+        let afterVersion = FingerprintJSVersion.v1
```

Comment:
To me `after` prefix suggests that the version should be excluded from the result, which is not the case with `.since(_:)` function.
```suggestion
        let fromVersion = FingerprintJSVersion.v1
```